client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
```

Comment:
I'm sort of struggling to understand what this means.
In the comment components I assumed the email field was a string (we show it in the comment author details), but testing with real data I've noticed the props typechecking fails here.
What's the reason behind a boolean email field?